### ISSUE TITLE
[Issue] Maximize Window [Request] Fullscreen

### ISSUE THREAD

**Issue by vslmc** [digimezzo/dopamine-windows#408]
Sine build build 912 or 970 the windows doesnt maximaze to the full screen, it shows a 1px line on the bottom
a screenshot (maybe you have to zoom in): 

![dopamine](https://cloud.githubusercontent.com/assets/28356741/25684836/dff13a86-3063-11e7-8a9c-dc535e67196f.png)

And I have a feature request: 
A fullscreen modus, toggle with F11 (alt+enter)
Maybe like a windows app, and only hidding the taskbar

**Comment by amaan6999** [comment]
![screenshot 100](https://cloud.githubusercontent.com/assets/28360239/25675134/0b2aa572-3003-11e7-8229-f97fc99a240e.png)
 The same issue as shown in the image.

**Comment by Magentaize** [comment]
<del>I get it, due to the fix for users who use auto-hide taskbar, their taskbar cannot re-appear when dopamine is maximized, so I removed 1px. I'll fix it for the default user.</del>

Update: Due to the bug of WindowChrome in WPF, it cannot be fixed (behaves as same as MahApp.Metro and Steam).

**Comment by mchowdhury** [comment]
@PeoLeser Are you sure? This bug was only introduced in the latest preview. Before that this bug did not exist?

**Comment by digimezzo** [comment]
@PeoLeser I didn't look into detail, but I do think this bug is related to the TaskBar fix. It didn't occur before. Do you mean we have to choose: either the taskbar cannot re-appear, or there is a 1px space at the bottom? It's not possible to fix the 1px space without breaking the taskbar again?

**Comment by amaan6999** [comment]
Yeah, should have mentioned. This is the latest preview version. Don't remember this problem with builds way earlier tho. Also, my taskbar is locked in. I don't set it to auto-hide.

**Comment by Magentaize** [comment]
@digimezzo To fix this issue, detecting the state of auto-hide of taskbar can be implemented, if users who doesn't set it to auto-hide the border keep original appearance, but for those who sets auto-hide, this 1px will still be removed and follow the taskbar's direction. It's a compromise for users who uses auto-hide taskbar, to re-appear it, 1px is necessary. I have modified the logic in pull request of WPFControl.
I reviewed the code of Intel graphics control center and Steelseries Engine, they don't have this problem because they haven't used WindowChrome.
